src/ray/raylet/worker_pool.cc

```diff
@@ -981,6 +982,9 @@ void WorkerPool::PushWorker(const std::shared_ptr<WorkerInterface> &worker) {
     int64_t now = get_time_();
     idle_of_all_languages_.emplace_back(worker, now);
     idle_of_all_languages_map_[worker] = now;
+  } else if (!found) {
```

Comment:
Have you actually seen this case? Should we create an issue?

Reply:
https://github.com/ray-project/ray/issues/32271

Reply:
nice catch!